docs/GHA.md

```diff
@@ -1,60 +1,66 @@
 # GitHub Actions
 
-The Buildkite Migration tool's currently supported, partially supported and unsupported properties in translation GitHub Actions workflows to Buildkite pipelines are listed below:
-
-## Supported properties
-
-### env 
-
-GitHub Actions allows the specification of environment variables at the workflow level, at a `job` level, and for each `step` within a `job`.
-
-Environment variables that are defined at the top of a workflow will be transition to [build level](https://buildkite.com/docs/pipelines/environment-variables#environment-variable-precedence) environment variables in the generated Buildkite pipeline. Environment variables defined within the context of each of a workflow's `jobs` are transitioned to [step level](https://buildkite.com/docs/pipelines/environment-variables#runtime-variable-interpolation) environment variables. Environment variables that are defined at `step` level are translated as a variable definition within the `commands` of a Buildkite [command step](https://buildkite.com/docs/pipelines/command-step).
+The Buildkite Migration tool's currently supported (🟢), partially supported (🟡) and unsupported (🔴) properties in translation of GitHub Action workflows to Buildkite pipelines are listed below.
 
 > [!NOTE]  
 > The Buildkite Migration tool does not currently support [GitHub secrets](https://docs.github.com/en/actions/security-guides/using-secrets-in-github-actions) stored within GitHub organizations or repositories (such as `{{ secrets.FOO }}`). For securely storing secrets that can be used within a Buildkite pipeline - the [S3 Secrets Buildkite plugin](https://github.com/buildkite/elastic-ci-stack-s3-secrets-hooks) can be setup/installed within a Buildkite agent (installed automatically within [Elastic CI Stack](https://github.com/buildkite/elastic-ci-stack-for-aws) setups). Additionally, Buildkite supports using a [secret storage service](https://buildkite.com/docs/pipelines/secrets#using-a-secrets-storage-service) through the use of dedicated plugins such as [HashiCorp Vault](https://github.com/buildkite-plugins/vault-secrets-buildkite-plugin) and [AWS Systems Manager Parameter Store](https://github.com/buildkite-plugins/aws-ssm-buildkite-plugin).
```

Comment:
I know that you have not changed this notice, but maybe change its wording to the fact that secret handling in BK is different and, instead of being part of the platform itself, for security reasons relies on plugins and external providers

Reply:
Adjusted; primarily focused on secrets deliberately being decoupled from the core platform - and secrets service/plugin usage needed!